lib/import_users.rb

```diff
@@ -86,4 +94,24 @@ def create_user(username, password_digest, title, first_name, last_name, full_na
     ea.verified = true
     ea.save!
   end
+
+  def generate_username(first_name, last_name, email)
+    # Use first name and last name as the basis for generating username
+    base = "#{first_name} #{last_name}".strip
```

Comment:
is there a reason for the space between first and last name?  gets `gsub`bed out later.